testapps/test_pillowtop/tests/test_form_pillow.py

```diff
@@ -22,13 +30,15 @@ def setUp(self):
         super(FormPillowTest, self).setUp()
         FormProcessorTestUtils.delete_all_xforms()
         self.pillow = get_xform_pillow(skip_ucr=True)
-
+        self.es = get_es_new()
+        initialize_index_and_mapping(self.es, USER_INDEX_INFO)
```

Comment:
Does the index need to be recreated for every test? If not, I think this should go in `setUpClass` since it does not need to be run for every test. And `ensure_index_deleted(USER_INDEX)` should be moved to `tearDownClass`.

Reply:
Surely. I was not aware of the difference before. Thanks for pointing that out. I have made the changes.